onnx/defs/shape_inference.h

```diff
@@ -105,6 +105,8 @@ struct InferenceContext {
   virtual const SparseTensorProto* getInputSparseData(size_t index) const = 0;
   // Gets the shape inputs computed by partial data propagation.
   virtual const TensorShapeProto* getSymbolicInput(size_t index) const = 0;
+  // To display a name the user can use to narrow its search.
+  virtual std::string getDisplayName() const = 0;
```

Comment:
May be useful to provide a default implementation for backward compatibility (with other tools/implementations, like ORT). That way, they can add a custom implementation when they are ready.

Reply:
I mean even an empty string would be ok.

Reply:
True.

Reply:
done